pkg/osquery/table/mdm.go

```diff
@@ -66,6 +106,7 @@ func generateMDMInfo(ctx context.Context, queryContext table.QueryContext) ([]ma
 					"identity_certificate_uuid": enrollProfile.IdentityCertificateUUID,
 					"installed_from_dep":        depEnrolled,
 					"user_approved":             userApproved,
+					"has_dep_profile":           depCapable,
```

Comment:
That's a better name than "dep_capable" and IMO you should rename the other variables/struct fields in this code. All macs are "capable" of DEP. Some serial numbers are not possible to add due to geographic regions/legal but that information can't be derived from the device itself.